### ISSUE TITLE
Annotation property with name starting with 'is' for Play/Scala model class not output properly

### ISSUE THREAD

**Issue by maneeshsahu** [swagger-api/swagger-core#838]
This annotation:

````scala
@ApiModel("VerifyCert")
case class VerifyCert(
        @(ApiModelProperty @field)(position=1, required=true, value="Issuer Distinguished Name.") issuerDN: String,
```

is output as 

```js
"properties":{"suerDN":{"type":"string"}
```

Seems like a check is made for 'is' that is truncating the property name to 'suer'.

**Comment by fehguy** [comment]
I believe this has been addressed in 1.3.13-SNAPSHOT.  Can you please retry that version?

**Comment by maneeshsahu** [comment]
This app is on play 2.2.2 that is why we are using 1.3.7. Is there a SNAPSHOT for play 2.2.2?

**Comment by webron** [comment]
No, support for play 2.2.X has dropped since 1.3.9.

**Comment by fehguy** [comment]
@maneeshsahu you might be able to force update it to use swagger-core 1.3.13-SNAPSHOT.  can you give that a shot?

**Comment by webron** [comment]
@maneeshsahu - any update on this issue?

**Comment by fehguy** [comment]
this has been fixed in 1.5.x, and will be picked up automatically with the scala support.
